wicket-spring/src/main/java/org/apache/wicket/spring/injection/annot/AnnotProxyFieldValueFactory.java

```diff
@@ -117,18 +118,26 @@ public Object getFieldValue(final Field field, final Object fieldOwner)
 		{
 			SpringBean annot = field.getAnnotation(SpringBean.class);
 
-			String name;
-			boolean required;
+			String name = null;
+			Boolean required = true;
```

Comment:
I would declare this one as `boolean` it seems to be not nullable :)

Reply:
Good idea, thanks